test/features/georoutingv2/georoutingv2.test.js

```diff
@@ -547,6 +548,17 @@ describe('GeoRouting', () => {
     expect(document.querySelector('.picker')).to.be.null;
   });
 
+  it('Add class .top to picker when there is no space to render below the trigger button', async () => {
+    window.innerHeight = 200;
```

Comment:
It'd be better to use the `setViewport` method from the web test runner

Reply:
fixed